multiqc/modules/kat/kat.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from KAT """
+import logging
+from collections import OrderedDict
+import json
+
+from multiqc.modules.base_module import BaseMultiqcModule
+from multiqc.plots import table
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='KAT', anchor='kat',
```

Comment:
Maybe set the `name` to `K-mer Analysis Toolkit` instead of `KAT`? There is probably space for it and it's a little more descriptive..